pkg/sql/logictest/logic.go

```diff
@@ -611,6 +614,13 @@ var logicTestConfigs = []testClusterConfig{
 		overrideDistSQLMode: "on",
 		overrideAutoStats:   "false",
 	},
+	{
```

Comment:
why `fakedist`? I'd really like to see a regular `local` variant with a single node.